docs/azure_container_apps.md

```diff
@@ -0,0 +1,22 @@
+# Deploying on Azure Container Apps
+
+Due to [a limitation](https://github.com/Azure/azure-dev/issues/2736) of azd, the azure.yaml file for deploying to Azure Container Apps lives here.
+To deploy to azure container apps, please run from project root folder:
```

Comment:
I'm nervous about having a .azure folder in a subfolder, as we have a lot of code that tries to get the azd env values, like the VS Code launch.json configuration. It won't work if we can't rely on knowing where the azd env file is. Isn't there also an option of "azd up --cwd" that would keep the .azure in the main folder?

Reply:
It doesn't seem to work.
![image](https://github.com/user-attachments/assets/4d169bd0-dcc1-4d07-8f28-74654b34c46e)

Besides `azd env get-values` doesn't honor `--cwd` and only picks up .azure from the current cwd.

Reply:
Added aca as a commented out host option, which should place env files in .azure in project root folder.